src/plugins/tables/tables.js

```diff
@@ -147,6 +153,19 @@ var componentName = "wb-tables",
 // Bind the init event of the plugin
 $document.on( "timerpoke.wb " + initEvent, selector, init );
 
+// Bind the update event of the plugin
+$document.on( updateEvent, selector, function( event ) {
+	var $elm = $( event.currentTarget );
+
+	$elm.DataTable().destroy();
+
+	if ( event.newData ) {
```

Comment:
If you send data using `data-wb-tables` and then want to modify this data and add new one, you would use this.